food-and-agriculture-organization-of-the-united-nations.csl

```diff
@@ -10,196 +10,368 @@
     <author>
       <name>Bin Liu</name>
       <email>lieubean@gmail.com</email>
+      <uri>https://www.linkedin.com/in/lieubean/</uri>
     </author>
+    <contributor>
+      <name>Julian Plummer</name>
+      <email>Julian.Plummer@fao.org</email>
+    </contributor>
     <category citation-format="author-date"/>
     <category field="science"/>
     <category field="social_science"/>
-    <summary>This style is created to meet the citation and bibliographical requirements of FAO House Style.</summary>
-    <updated>2015-08-24T12:56:02+00:00</updated>
+    <summary>This style is created to meet the citation and bibliographical requirements of FAOSTYLE, and has been tested with Zotero and Mendeley. Last update: March 2017.</summary>
+    <updated>2017-03-08T16:33:05+01:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
+  <!-- Locale settings for English. Settings for other languages could be added later to this section. -->
   <locale xml:lang="en">
+    <style-options punctuation-in-quote="false"/>
     <terms>
       <term name="editor" form="verb-short">ed. by</term>
       <term name="editor" form="short">
```

Comment:
If both forms are going to be without period, just remove the terms and use strip-periods="true" in the style for simplicity.